react/containers/ChannelCreateForm/view.jsx

```diff
@@ -107,7 +90,7 @@ class ChannelCreateForm extends React.Component {
         this.props.onChannelLogin(result.channelName, result.shortChannelId, result.channelClaimId);
       })
       .catch((error) => {
-        this.setState({'error': error.message, status: null});
+        if (error.message) this.setState({'error': error.message, status: null});
```

Comment:
Always use brackets for if statements IMO.

Also - is Prettier and/or a linter not set up for this project? It can catch and even fix ones like this.

Reply:
yes to linter, not yet to Prettier (#375)